merkle/memory_merkle_tree_test.go

```diff
@@ -246,23 +274,39 @@ func referenceSnapshotConsistency(inputs [][]byte, snapshot2 int64,
 	if snapshot1 <= split {
 		// Root of snapshot1 is in the left subtree of snapshot2.
 		// Prove that the left subtrees are consistent.
-		subproof = referenceSnapshotConsistency(inputs[:split], split, snapshot1,
+		s, err := referenceSnapshotConsistency(inputs[:split], split, snapshot1,
 			treehasher, haveRoot1)
```

Comment:
nit/pre-existing: move params to line above.

Reply:
done